README.md

```diff
@@ -85,48 +83,37 @@ Device         Boot   Start      End  Sectors  Size Id Type
-For this purpose, the following configuration variables are used:
-
-- `OMNECT_PART_OFFSET_UBOOT_ENV1`
-    - offset of 1st u-boot environment bank (in KiB, decimal)
-- `OMNECT_PART_OFFSET_UBOOT_ENV2`
-    - offset of 2nd u-boot environment bank (in KiB, decimal)
-- `OMNECT_PART_SIZE_UBOOT_ENV`
-    - size of one u-boot environment bank (in KiB, decimal)
+**Note1**: The size of `data` depends on your sdcard/emmc/nvme size. Per default it has a size of 512M and is resized on the first boot to the max available size.<br>
+**Note2**: The size of `rootA` and `rootB` depends on your board and image variant (development or release). Concrete sizes can be found in the board specific [readme](./doc/board/).<br>
+**Note3 (only for images where u-boot is used as OS bootloader)**: There is a reserved area between the boot partition and the rootA partition used for two redundant u-boot environment banks. For this purpose, the following configuration variables are used:
+- `OMNECT_PART_OFFSET_UBOOT_ENV1`: offset of 1st u-boot environment bank (in KiB, decimal)
+- `OMNECT_PART_OFFSET_UBOOT_ENV2`: offset of 2nd u-boot environment bank (in KiB, decimal)
+- `OMNECT_PART_SIZE_UBOOT_ENV`: size of one u-boot environment bank (in KiB, decimal)
 
 ## Compatibility
 `meta-omnect` is compatible with the current yocto LTS release branch `kirkstone`.
 
 ## Supported Devices
-See [README.device.md](./README.device.md).
+See [BSP-FEATURE-OVERVIEW.md](./doc/board/BSP-FEATURE-OVERVIEW.md).
 
 ## Versioning
 We reflect the used poky version in our version schema. `omnect-os` is versioned via `POKY_VERSION.BUILD_NR`, `4.0.x.y` where `x` is poky kirkstone's patch version and `y` is the build number.
 
 ## Dependencies
 `meta-omnect` depends on:
-- mandatory:
-    - [meta-openembedded](https://github.com/openembedded/meta-openembedded.git): `meta-filesystems`, `meta-networking`, `meta-oe` and `meta-python`
-    - [meta-security] (https://git.yoctoproject.org/meta-security)
-    - [meta-swupdate](https://github.com/sbabic/meta-swupdate.git)
-    - [meta-virtualization](https://git.yoctoproject.org/meta-virtualization)
-    - [poky](https://git.yoctoproject.org/poky)
-- optional:
-    - [meta-phytec](https://github.com/phytec/meta-phytec) (optional - via dynamic layer, phytec polis support depends on it)
-    - [meta-freescale](https://github.com/Freescale/meta-freescale) (optional - via dynamic layer, phytec polis support depends on it)
-    - [meta-raspberrypi](https://github.com/agherzan/meta-raspberrypi.git) (optional - via dynamic layer, raspberrypi support depends on it)
-
+  - [meta-openembedded](https://github.com/openembedded/meta-openembedded.git): `meta-filesystems`, `meta-networking`, `meta-oe` and `meta-python`
+  - [meta-security](https://git.yoctoproject.org/meta-security)
+  - [meta-swupdate](https://github.com/sbabic/meta-swupdate.git)
+  - [meta-virtualization](https://git.yoctoproject.org/meta-virtualization)
+  - [poky](https://git.yoctoproject.org/poky)

```

Comment:
Do you think that's necessary? I mean we provide the links to the repo and clicking the respective readme is sth. the reader should be able to do without a hint from us.

Reply:
actually, it should be clear for a yocto developer that this can't be the full truth, but the phrase "meta-omnect depends on:" suggests a final list imho.
call it nitpicking, but I'd prefer to make that clear, maybe by simply changing it to something along the lines of "aside from hardware specific meta layers meta-omnect depends on:"